src/Cms/ModelWithContent.php

```diff
@@ -702,7 +702,7 @@ public function update(array $input = null, string $languageCode = null, bool $v
             }
         }
 
-        $arguments = [static::CLASS_ALIAS => $this, 'values' => $form->data(), 'strings' => $form->strings(), 'languageCode' => $languageCode];
+        $arguments = [static::CLASS_ALIAS => $this, 'values' => $form->content(), 'strings' => $form->strings(), 'languageCode' => $languageCode];
```

Comment:
By using `$form->content()` here, this becomes a breaking change, right? What was the idea behind using it here? I'm not entirely sure I can follow.

Reply:
I don't think it's a breaking change, actually `\Kirby\Data\Txt` handler is already deleting `null` values (other handlers don't). So we send to handler cleared data from `null` values.

If we think there will be a problem, `$form->content()` method will be sufficient to use it only in the `structure` field.

Reply:
The $form->content() method does no longer contain the null values though, right? It's the one that returns the array with deleted keys for fields that should not be stored. That's why I think it would be a breaking change unless I missed something.

Reply:
If I understand it correctly;

If a field is deleted from blueprint, its data continues to be kept in the txt file. This PR only concerns fields that cannot be saved as `save: false`.

As I said, it will be sufficient to use this method only in the structure field if you want.

Reply:
I think I would be better to focus on the structure field for now in this PR.